jolly-jellyfish/src/page_maker/views.py

```diff
@@ -1,3 +1,219 @@
-from django.shortcuts import render
+from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
+from django.shortcuts import get_object_or_404
+from django.urls import reverse_lazy
+from django.views.generic import (
+    TemplateView,
+    FormView,
+    DetailView,
+    UpdateView,
+    DeleteView,
+    ListView
+)
 
-# Create your views here.
+from .forms import UserRegisterForm, WebpageForm, TemplateForm, CommentForm
+from .models import User, Webpage, Template, Comment
+
+
+class MainView(TemplateView):
+    template_name = "page_maker/main.html"
+
+
+class UserRegister(FormView):
+    template_name = 'registration/register.html'
+    form_class = UserRegisterForm
+    success_url = '/users/login'
+
+    def form_valid(self, form):
+        form.save()
+        return super().form_valid(form)
+
+
+class UserDetailView(DetailView):
+    model = User
+    context_object_name = 'viewed_user'
+    template_name = 'page_maker/user_detail.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        # user = get_object_or_404(User, pk=self.kwargs.get('pk'))
```

Comment:
Useless commented line?

Reply:
I'm not sure, @baru64 might be reminding himself of something. It's also possible that this is already removed on his branch.